src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      class="big-form"
+      method="post"
+      data-qa="big">
+      <label class="search-big">
+        <img
+          class="icon-big"
+          src="./images/Search.svg"
+           alt="search-big">
+        <input
+            class="big-input"
+            type="text"
+            data-da="keypress"
+            placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
+      action="/"
+      class="small-form"
+      method="post"
+      data-qa="small">
+      <label class="search-small">
+        <img
+          class="icon-small"
+          src="./images/Search.svg"
+           alt="search-small">
+        <input
+            class="small-input"
+            type="text"
+            placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+
+    </form>
   </body>
```

Comment:
```suggestion
    <form
      action="/"
      class="form"
      method="post"
      data-qa="big">
      <label class="search">
        <img
          class="icon"
          src="./images/Search.svg"
           alt="search-big">
        <input
            class="input"
            type="text"
            data-da="keypress"
            placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
    </form>
    <form
      action="/"
      class="form small-form"
      method="post"
      data-qa="small">
      <label class="search search-small">
        <img
          class="icon icon-small"
          src="./images/Search.svg"
           alt="search-small">
        <input
            class="input small-input"
            type="text"
            placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>

    </form>
  </body>
```
lets make it work like this

Reply:
Виправив